server/src/main/java/org/elasticsearch/action/synonyms/GetSynonymsAction.java

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.action.synonyms;
+
+import org.elasticsearch.action.ActionRequest;
+import org.elasticsearch.action.ActionRequestValidationException;
+import org.elasticsearch.action.ActionResponse;
+import org.elasticsearch.action.ActionType;
+import org.elasticsearch.common.io.stream.StreamInput;
+import org.elasticsearch.common.io.stream.StreamOutput;
+import org.elasticsearch.synonyms.SynonymRule;
+import org.elasticsearch.synonyms.SynonymsManagementAPIService;
+import org.elasticsearch.xcontent.ToXContentObject;
+import org.elasticsearch.xcontent.XContentBuilder;
+
+import java.io.IOException;
+import java.util.Objects;
+
+import static org.elasticsearch.action.ValidateActions.addValidationError;
+
+public class GetSynonymsAction extends ActionType<GetSynonymsAction.Response> {
+
+    public static final GetSynonymsAction INSTANCE = new GetSynonymsAction();
+    public static final String NAME = "cluster:admin/synonyms/get";
+
+    public GetSynonymsAction() {
+        super(NAME, Response::new);
+    }
+
+    public static class Request extends ActionRequest {
+        private final String SynonymsSetId;
```

Comment:
we always start variable names with small letters, should we rename this to `synonymsSetId`?